src/app/page/Aboutus/Aboutus.js

```diff
@@ -1,46 +1,78 @@
-import { Box, Divider, Typography } from "@material-ui/core";
+import { Grid } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
 import React from "react";
+import { Link } from "react-router-dom";
 
+import AboutImage1 from "../../../img/AboutImage1.webp";
```

Comment:
Asking for cross browser development requirements because according to [caniuse](https://caniuse.com/#feat=webp), webp image formats aren't available on safari and IE.

You can use web optimization tools to scale images down to lower resolution needed for web. PNG and JPG should be fine, so long as our image is scaled down for the views we need.